### PR TITLE
VATUSA Traning system

### PR DESCRIPTION
**Pull request by Frafire**
## [ZAB]

---

### Summary

What does this PR do?
Push the new submiting system in development branche ready for the release.
---

### DB Changes (if any)

- [] What is the change?

```
changes here
```

---

### Testing

1. Run traning test

---

### Post-Deployment Validation

How to verify in the production environment.
verify the submission
---

### CODE REVIEW

File: .github/workflows/gf-test-deploy.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+name: Build and Deploy to Beta Site
```
Comment (cdconn00) [inline .github/workflows/gf-test-deploy.yml:1]: Delete this file now that we don't need the pipeline

File: src/views/instructor/training/EditSession.vue [hunk 2]
```diff
@@ -180,28 +182,42 @@ export default {
...
+				// formData.append('session_date', this.session.startTime);
+				// formData.append('position', this.session.position);
+				// formData.append('duration', this.duration);
+				// formData.append('movements', this.session.movements);
+				// formData.append('score', this.session.progress);
+				// formData.append('notes', this.session.studentNotes);
+				// formData.append('location', this.session.location);
+				// formData.append('ots', this.session.ots);
+				// await vatusaApiAuth.post(`/user/${this.session.student.cid}/training/record`, formData);
+
```
Comment (cdconn00) [inline src/views/instructor/training/EditSession.vue:202]: Remove commented-out lines.

File: src/views/instructor/training/EditSession.vue [hunk 3]
```diff
@@ -180,28 +182,42 @@ export default {
...
+					"location": this.session.location,
+					"ots": this.session.ots
+				});
+				
 
 				// VATUSA API response
 				this.toastSuccess('Traning Record issued');
 			} catch(e) {
 				this.toastError(e);
+				console.log(e);
```
Comment (cdconn00) [inline src/views/instructor/training/EditSession.vue:220]: Do we need this now that the feature is done?

File: src/views/instructor/training/EditSession.vue [hunk 4]
```diff
@@ -180,28 +182,42 @@ export default {
...
+					"movements": this.session.movements,
+					"score": this.session.progress,
+					"notes": this.session.studentNotes,
+					"location": this.session.location,
+					"ots": this.session.ots
+				});
+				
 
 				// VATUSA API response
 				this.toastSuccess('Traning Record issued');
```
Comment (cdconn00) [inline src/views/instructor/training/EditSession.vue:217]: Change to _Training Record Issued_ (consistent capitalization)

**Review by cdconn00 — CHANGES_REQUESTED** [review]
See cleanup changes requested.

**Comment by cdconn00** [comment]
@Frafire Please resolve the merge conflict,

File: src/views/instructor/training/EditSession.vue (hunk 3 above)
Comment (Frafire) [reply src/views/instructor/training/EditSession.vue:220]: no, i delete it in new version

**Review by cdconn00 — APPROVED** [review]

**Comment by cdconn00** [comment]
@Frafire appvd, verify working in dev then ask JW to give you the green light before going to production @JWZABARTCC

**Pull request merged**